README.md

```diff
@@ -7,6 +7,17 @@
 described by:
  http://www.sqlite.org/fileformat.html
 
+## Current status
+
+[![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/)
```

Comment:
I wouldn't perhaps go as far as say it's unmaintained.
perhaps just slap a big "Work In Progress / WIP" ?

Reply:
:-) I just really like that badge… fixed.